example/ck_tile/03_gemm/gemm_basic.cpp

```diff
@@ -110,12 +111,32 @@ int run_gemm_example(int argc, char* argv[])
     using Row = ck_tile::tensor_layout::gemm::RowMajor;
     using Col = ck_tile::tensor_layout::gemm::ColumnMajor;
 
+    std::string precision = arg_parser.get_str("prec");
```

Comment:
can we change precision name to data_type?

Reply:
Done. Changed the code accordingly.